### ISSUE TITLE
Multilanguage support for chromevox

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/google-axs-chrome#25]
```
Chromevox only supports English language. Some tokens like links, headers, list 
groups and more use words in english. Many users have speech engines for other 
languages than english and many of those users can't understand these tokens in 
english.

Chromevox should support more languages than english.
I can translate a template for spanish
```

Original issue reported on code.google.com by `Tyflos2...@gmail.com` on 4 Oct 2011 at 7:11

**Comment by GoogleCodeExporter** [comment]
```
You've raised a good point, and actually we are working on multilanguage 
support for ChromeVox. We should have an update soon.
```

Original comment by `rshea...@google.com` on 4 Oct 2011 at 4:36
- Changed state: **Started**
- Added labels: **ChromeVox**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.18.0. http://www.chromevox.com/release_notes.html
```

Original comment by `rshea...@google.com` on 14 Feb 2012 at 9:42
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
How about Chrome OS and support for other languages when using chromevox? Only 
found US english only extensions. 
```

Original comment by `dennisia...@gmail.com` on 18 May 2013 at 11:34
